frontend/src/navbar.tsx

```diff
@@ -35,6 +36,11 @@ import {
   faCircleExclamation,
 } from "@fortawesome/free-solid-svg-icons";
 
+// Set the page title
+useEffect(() => {
```

Comment:
`useEffect` and other react hooks can only be used *inside* a react componet.

I'd do something like (inside the component):

```
const title = getTitle();

useEffect(() => document.title = title, [title]);
```

Reply:
Updated via https://github.com/iFixit/pulldasher/pull/363/commits/46167a89e2871f861d876050a1e9f312db4335c1